apps/examples/testcase/le_tc/kernel/tc_pthread.c

```diff
@@ -1539,6 +1540,58 @@ static void tc_pthread_pthread_setcanceltype(void)
 }
 #endif
 
+/**
```

Comment:
`pthread_mutex_consistent` API has dependency with `CONFIG_PTHREAD_MUTEX_UNSAFE`.
So we should enclose this tc with `#ifndef CONFIG_PTHREAD_MUTEX_UNSAFE`.